### ISSUE TITLE
Make `ball_radius` a `ParsedFunction` in tutorial step 6b

### ISSUE THREAD

**Issue by jwpeterson** [idaholab/moose#5294]
We set up a `LinearInterpolation` object in this step to relate ball radius and permeability... and only use it for a single value of ball radius.  In part b of this step, we should provide a `ParsedFunction` which lets the ball radius vary in either x or y.

**Comment by friedmud** [comment]
Actually, the plan was to read ball_radius from an auxiliary variable...
and then eventually even have that be dynamic based on the temperature
(because the balls will swell).

**Comment by permcody** [comment]
Steel balls swelling? While this sounds interesting, I can't imagine that
this scenario will subject the balls to extreme temperatures to the point
where the swelling is much of an issue. I guess this is a demo though so
the capability would be cool.

**Comment by friedmud** [comment]
Yep, just for demo purposes.  We could even solve for the swelling of the
balls as a 1D-spherical sub-app using MultiApps and transfer the new radius
back to the master app as a field to demonstrate MultiApps and Transfers.

**Comment by aeslaughter** [comment]
You just want to say "ball swelling" in the training, which I am all for.

**Comment by friedmud** [comment]
And then when they cool off we can call it "shrinkage"  ;-)
